deployments/helm-chart/values.yaml

```diff
@@ -310,3 +310,6 @@ prometheus:
 
   ## Configures the port to scrape the metrics.
   port: 9113
+
+  ## Specifies the namespace/name of a Kubernetes TLS Secret which will be used to protect the Prometheus endpoint.
+  secret: ""
```

Comment:
there are two docs for helm chart that we need to update to include the new parameters:
- https://github.com/nginxinc/kubernetes-ingress/blob/master/deployments/helm-chart/README.md
- https://deploy-preview-1652--nginx-kubernetes-ingress.netlify.app/nginx-ingress-controller/installation/installation-with-helm/

Reply:
I've updated the first link but I couldn't find the second in the codebase.

Reply:
here is the link https://github.com/nginxinc/kubernetes-ingress/blob/master/docs-web/installation/installation-with-helm.md

Reply:
thanks, updated.